NYPLCardCreator/Classes/UserSummaryViewController.swift

```diff
@@ -0,0 +1,239 @@
+import UIKit
+
+/// This class is used for summarizing the user's details before
```

Comment:
This is extremely anal, but this comment should end with a period to be consistent with Apple's conventions for class descriptions.